packages/devextreme/js/__internal/grids/new/card_view/headers/view.tsx

```diff
@@ -25,6 +26,16 @@ export class HeadersView extends View {
     super();
   }
 
+  public onHeaderCloseButtonClick(name: string): void {
+    const index = this.getColumnIndexByName(name);
+
+    this.columnsController.columns.updateFunc((columns) => {
+      const newColumns = columns.slice();
```

Comment:
As I understood, you want to make a copy of array, if so than let's use more readable version with spread operator.
`columns.slice()` -> `[ ...columns ]`

Reply:
If we want to use unified approach, let's discuss separately and apply strict rules to it (like linter)

Reply:
It's a question of the habbit or personal preferences but historically slice method was used as a quick workaround/hack to make a clone of array because there were no other eloquent ways in JS standard. For me it's like a "Hello from 2015".

Today, there are many ways how to make a copy of array:
- for loop
- slice
- Array.from
- spread operator

I suggest to use modern approachs like Array.from or spread operator and avoid hacks old approaches. Agree that we need to add a rule to the linter.